### ISSUE TITLE
Doctests creates several files

### ISSUE THREAD

**Issue by raphaeltimbo** [ross-rotordynamics/ross#590]
When we run the doctests they create several files in the ROSS directory, this is annoying since these documents can sometimes go into commits by mistake.
Ideally doctests would not create or modify files.
I was looking for solutions to this in numpy docstrings and I saw that they use a temporary file in this case:
```python
    Examples
    --------
    >>> from tempfile import TemporaryFile
    >>> outfile = TemporaryFile()
    >>> x = np.arange(10)
    >>> np.save(outfile, x)
```

Another option would be to add these files to a /tmp/ folder that could be added to the project's .gitignore file:
```python
    Examples
    --------
    Store data to disk, and load it again:
    >>> np.save('/tmp/123', np.array([[1, 2, 3], [4, 5, 6]]))
    >>> np.load('/tmp/123.npy')
    array([[1, 2, 3],
           [4, 5, 6]])
```
I think the second solution is better, since the import from TemporaryFile might cause some confusion.

**Comment by rodrigomoliveira1** [comment]
The second solution seens better to me too.

**Comment by raphaeltimbo** [comment]
I have implemented the second solution and we have merged the code in #601. 
The problem is that appveyor was not working at that time (#611) and we didn't notice that the code does not work on windows.
I will implement the solution based on the tempfile module (https://stackoverflow.com/a/847866/5726899).
